src/stdatamodels/jwst/datamodels/schemas/miri_mrsptcorr.schema.yaml

```diff
@@ -9,27 +9,16 @@ allOf:
 - $ref: keyword_pexptype.schema
 - type: object
   properties:
-    data:
-      title: Photometric leak array
-      fits_hdu: SCI
-      default: 1.0
-      ndim: 2
-      datatype: float64
-    err:
-      title: Uncertainty array
-      fits_hdu: ERR
-      default: 0.0
-      ndim: 2
-      datatype: float64
-    dq:
-      title: Data quality array
-      fits_hdu: DQ
-      default: 0
-      ndim: 2
-      datatype: uint32
-- $ref: dq_def.schema
-- type: object
-  properties:
+    leakcor_table:
+      title: IFU spectral leak correction
+      fits_hdu: LEAKCOR
+      datatype:
+      - name: wavelength
+        datatype: float32
+      - name: frac_leak
```

Comment:
Not a big deal, but unlike FITS keyword names, we don't need to be stingy with our table column names and hence you could (if you want) spell these out more completely, for readability. Same for `err_leak`.

Reply:
I'd be inclined to leave it as-is, for consistency with a draft file that's been in offline use via Box for a couple weeks

Reply:
That's fine. Like I said, it's not a big deal.